src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheck.java

```diff
@@ -295,7 +295,9 @@ public void visitToken(DetailAST ast) {
             final TextBlock textBlock =
                 contents.getJavadocBefore(ast.getFirstChild().getLineNo());
 
-            checkComment(ast, textBlock);
+            if (textBlock != null) {
+                checkComment(ast, textBlock);
```

Comment:
this cause testing coverage problem:
```
[INFO] --- jacoco:0.8.13:check (default-check) @ checkstyle ---
[INFO] Loading execution data file C:\projects\checkstyle\target\jacoco.exec
[INFO] Analyzed bundle 'checkstyle' with 841 classes
[WARNING] Rule violated for class com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck: branches covered ratio is 0.98, but expected minimum is 1.00
```

lets revert this IF and make:
`private void checkComment(final DetailAST ast, final @NUllable TextBlock comment) {`